hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AzcopyHelper.java

```diff
@@ -41,7 +43,7 @@ public class AzcopyHelper {
     private PrefixMode mode;
 
     public AzcopyHelper(String accountName, String fileSystemName, Configuration configuration ,PrefixMode mode) throws Exception {
-      this.accountName = accountName;
+      this.accountName = accountName.replace(ABFS_DNS_PREFIX, WASB_DNS_PREFIX);
```

Comment:
There was an issue recently where it was replacing the "dfs" with "blob" even in the account name itself if present. Can the same happen here as well??

If yes, we can handle it by replace(".dfs.", ".blob.")

Reply:
Resolved